kuma/core/helpers.py

```diff
@@ -150,8 +151,11 @@ def entity_decode(str):
 
 
 @register.function
-def inlinei18n(locale):
-    return StaticI18nJob().get(locale)
+def statici18n(locale):
+    if not locale:
+        locale = settings.LANGUAGE_CODE
+    filename = get_filename(locale, settings.STATICI18N_DOMAIN)
+    return static(os.path.join(settings.STATICI18N_OUTPUT_DIR, filename))
```

Comment:
Yep, exactly. We would need to run `compilejsi18n` before `collectstatic`. Then we could use `staticfiles_storage.url` on this line to use the staticfiles-managed asset.